src/argilla/client/sdk/v1/datasets/api.py

```diff
@@ -201,12 +201,9 @@ def get_records(
         A `Response` object containing a `parsed` attribute with the parsed response if the
         request was successful, which is an instance of `FeedbackRecordsModel`.
     """
-    url = f"/api/v1/datasets/{id}/records"
+    url = f"/api/v1/datasets/{id}/records?offset={offset}&limit={limit}&include=responses&include=suggestions"
```

Comment:
why this change?

Reply:
Because otherwise from the Python client we cannot define a Python dict with two keys named equally i.e. `{"include": "responses", "include": "suggestions"}`

Reply:
You can pass a list as value. See https://www.python-httpx.org/quickstart/

```python
>>> params = {'key1': 'value1', 'key2': ['value2', 'value3']}
>>> r = httpx.get('https://httpbin.org/get', params=params)
>>> r.url
URL('https://httpbin.org/get?key1=value1&key2=value2&key2=value3')
```